### DISCUSSION TITLE
Stirling-PDF; Uprade to v0.35; Error in the menu due to an incorrectly installed program

### DISCUSSION THREAD

**Discussion by roswitina** [community-scripts/ProxmoxVE#590] [General]
I have upgraded Stirling PDF from v0.32 to v0.35 with the script. Afterwards I noticed that I could no longer find the menu item listed: “Convert Pdf to Pdf/A”

A request to Stirling - PDF via Discord revealed that I have to uninstall ocrmypdf and reinstall qpdf. After I did that, the menu item was back and everything worked again.

**Comment by vhsdream** [comment]
I can confirm this. And I wanted to check if this happened in v0.35 or earlier, like 0.33 or 0.34.

v0.34
![stirling-0 34](https://github.com/user-attachments/assets/3c731544-434d-47c8-b411-c02fb520a051)

v0.35
![stirling-0 35](https://github.com/user-attachments/assets/d3496123-207c-482d-bde3-53a2dc9a2c98)

**Comment by vhsdream** [comment]
Tested a fix for this, made a PR - #614
